metrics/src/main/java/tech/pegasys/pantheon/metrics/MetricCategory.java

```diff
@@ -13,6 +13,7 @@
 package tech.pegasys.pantheon.metrics;
 
 public enum MetricCategory {
+  EVM("evm"),
```

Comment:
Should new categories go at the end of the list?  If not should we do a more sensible ordering like alphabetization?

Reply:
Alphabetical it is.